testbed/integration/data_providers.go

```diff
@@ -0,0 +1,49 @@
+package integration
+
+import (
+	"sync/atomic"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/testbed/testbed"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+var _ testbed.DataProvider = &sampleConfigsDataProvider{}
+
+type sampleConfigsDataProvider struct {
+	traces             ptrace.Traces
+	dataItemsGenerated *atomic.Uint64
+}
+
+// GenerateLogs implements testbed.DataProvider.
```

Comment:
Just a nitpick, but I would say we should remove these comments since we check that it implements the interface above, and that should be sufficient to determine what this method does.

Reply:
Alright I will remove. I mostly based what's in the original `testbed` package.